jvm-runtime/kotlin/scaffolding/{{ .Name | camel | lower }}/src/main/kotlin/{{ .PackageDir }}/EchoVerb.kt

```diff
@@ -0,0 +1,9 @@
+package {{ .Group }}
```

Comment:
How is the FTL runtime going to know what module it is? Does it rely on a property from the pom or...?

Reply:
It will either use the value from the toml file, or fall back to the artifact id in the pom if the end var is not set. 

This is something I want to talk to you about when we are in Sydney next week.